modules/infinite-scroll/infinity.php

```diff
@@ -1272,10 +1271,6 @@ function query() {
 			$results['type'] = 'empty';
 		}
 
-		if ( is_customize_preview() ) {
```

Comment:
We need to maintain support for current WP -1 version, so until 4.8 is out we'll need to add a version check here.  

`if ( version_compare( $wp_version, '4.7', '<' ) && ... ) `